server-toss/.env.example

```
@@ -21,3 +21,4 @@ SHARE_DAILY_CHARGE_LIMIT=5
 AD_GROUP_ID_WHITELIST=
 # 콘솔 공유 리워드에 등록한 모듈 ID 콤마 구분 (moduleId가 곧 리워드 ID)
 SHARE_MODULE_ID_WHITELIST=
+EXTERNAL_API=toss|mock
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)